remote-bench-tps.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
```

Comment:
let's just merge this script into bench-tps.sh, the two scripts are close enough and 2 sucks